packages/hydrogen-react/src/CartProvider.stories.tsx

```diff
@@ -246,7 +246,7 @@ Default.args = {
   numCartLines: 30,
   /** A callback that is invoked when the process to create a cart begins, but before the cart is created in the Storefront API. */
   data: undefined,
-  /** A fragment used to query the Storefront API's [Cart object](https://shopify.dev/api/storefront/2023-10/objects/cart) for all queries and mutations. A default value is used if no argument is provided. */
+  /** A fragment used to query the Storefront API's [Cart object](https://shopify.dev/api/storefront/current/objects/cart) for all queries and mutations. A default value is used if no argument is provided. */
```

Comment:
This is incorrect. I think it needs to stay referencing the exact version. The reason being is devs can access multiple versions of the docs. They can specifically view the docs for 2023-10, and any links inside those docs should remain consistent with that version. If we change all of these references to `current`, then when viewing older docs, clicking a link will redirect them to the latest. Best case would be annoying and confusing. Worst case the link might not exist in the current version anymore.

Reply:
Right.